unstructured/ingest/main.py

```diff
@@ -423,10 +423,36 @@ def run(self):
     help="User principal name, usually is your Azure AD email.",
 )
 @click.option(
-    "--ms-onedrive-folder",
+    "--ms-onedrive-path",
     default=None,
     help="Folder to start parsing files from.",
 )
+@click.option(
+    "--ms-sharepoint-site",
+    default=None,
+    help="Sharepoint site url. Process either base url e.g https://[tenant].sharepoint.com \
+        or relative sites https://[tenant].sharepoint.com/sites/<site_name>.",
+)
+@click.option(
+    "--ms-sharepoint-path",
+    default="Shared Documents",
+    help="Path to start parsing files from. If the connector is to process all sites \
+    within the tenant this filter will be applied to all sites document libraries. \
+    Default 'Shared Documents'",
+)
+@click.option(
+    "--ms-sharepoint-all",
+    is_flag=True,
+    default=False,
+    help="Process all sites within the tenant. Requires for the app to be registered at a tenant \
+        level and a site url in https://[tenant]-admin.sharepoint.com.",
+)
+@click.option(
+    "--ms-sharepoint-pages",
+    is_flag=True,
+    default=False,
+    help="Process site pages.",
+)
```

Comment:
I suspect users would want pages by default. Maybe we invert this logic and call it `--ms-sharepoint-files-only`?

Reply:
we probably should be consistent both between ENV vars and args, so yes. but also we should probably be consistent across connectors. wondering if we should drop the ms prefix altogether and refactor onedrive and outlook to just use their own connector specific variables rather than sharing ms- creds. don't need to solve for this pr, but maybe we bump that in a soon to follow pr.